### PR TITLE
fix(infra): Fix the cog batch job to put record to dynamodb.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
This should fix the following problems, when insert and update records to dynamodb.

image
image

### CODE REVIEW

File: packages/_infra/src/serve/db.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const TileMetadataTableArn = {
     return cdk.Arn.format(
       {
         service: 'dynamodb',
-        region: '*',
```
Comment (blacha) [inline packages/_infra/src/serve/db.ts:32]: I dont think this will fix the problem, this was to allow other regions to write to dynamodb

Comment (Wentao-Kuang) [reply packages/_infra/src/serve/db.ts:32]: I see. I need to have a look what cause the problem. Just don't how to test this.

Comment (blacha) [reply packages/_infra/src/serve/db.ts:32]: where is it coming from, do you have the bundled index.js to see the stack trace?

Comment (Wentao-Kuang) [reply packages/_infra/src/serve/db.ts:32]: Yes, That's what I am thinking to get a bundle index.js first.

Comment (Wentao-Kuang) [reply packages/_infra/src/serve/db.ts]: image
This the place of error, I can't find where is the place to using this. 
However, I am sure ti broke at the place for puting item to dynamodb.

**Review by blacha — APPROVED** [review]

**Pull request merged**